Library/Homebrew/test/utils/github_spec.rb

```diff
@@ -6,10 +6,9 @@
 describe GitHub do
   describe "::search_code", :needs_network do
     it "queries GitHub code with the passed parameters" do
-      results = described_class.search_code(repo: "Homebrew/brew", path: "/",
-                                            filename: "readme", language: "markdown")
+      results = described_class.search_code(user: "Homebrew", path: "/", filename: "readme", extension: "md")
 
-      expect(results.count).to eq(1)
+      expect(results.count).to eq(29)
```

Comment:
Hmm, this looks like it's changed behaviour if the results count has increased by so much?

Reply:
I think we have 32 repositories in the Homebrew organisation, of which 3 are archived. This change retrieves `README.md` files (present at `/README.md`) from the 29 non-archived repositories rather than just from `Homebrew/brew`.

Reply:
This change retrieves README.md files (present at /README.md) from the 29 public repositories rather than just from Homebrew/brew.

Reply:
I think the previous test was more in line with real-world usage. It's more likely that we search a specific repo rather than all in the org.

Reply:
@hyuraku Please update this so the test no longer requires `eq(29)`, thanks!